core/controllers/admin.py

```diff
@@ -459,20 +459,24 @@ def _load_dummy_new_skill(self):
                 raise Exception(
                     'User does not have enough rights to generate data.')
             skill_id = skill_services.get_new_skill_id()
+            skill_name = 'Dummy Skill %s' % python_utils.UNICODE(random.getrandbits(32)) # pylint: disable=line-too-long
             skill = self._create_dummy_skill(
-                skill_id, 'Dummy Skill with 15 questions',
-                '<p>Dummy Explanation 1</p>')
+                skill_id, skill_name, '<p>Dummy Explanation 1</p>')
+            skill_services.save_new_skill(self.user_id, skill)
             for i in python_utils.RANGE(15):
                 question_id = question_services.get_new_question_id()
-                question_name = 'Question %s' % python_utils.UNICODE(i)
+                question_name = 'Question number %s %s' % (python_utils.UNICODE(i), skill_name) # pylint: disable=line-too-long
```

Comment:
Break after '('. Don't use the "disable" pragma.

Reply:
Done.